Applications/Core TLP.md

```diff
@@ -0,0 +1,69 @@
+## Introduction
+
+Currently the Node.js Foundation directly oversees the development of the core platform and associated working groups. The addition of Top Level Projects will predictably increase the workload on the TSC and so it makes sense to transfer these responsibilities to a Top Level Project concerned solely with the development of the platform.
+
+## History & Metrics
+
+Created by Ryan Dahl, now the largest programming ecosystem in the world.
+
+## Scope
+
+The Core TLP will have sole responsibility and discretion over the node.js project in the following areas:
```

Comment:
The original draft was written before we went "Universal Capitalization," fixing.